### ISSUE TITLE
Commands for creating an Account not working anymore, which are the new commands

### ISSUE THREAD

**Issue by feuerstein-dev** [ankicommunity/anki-devops-services#48]
Hello, i've set up an Anki Sync Server, with this Docker Image:
https://hub.docker.com/r/ankicommunity/anki-sync-server
Now i have the Problem, that i can enter the command line of the docker container, but the commands aren't working anymore. What are the new commands for the container?
It would be happy if someone could help me.
Thank you in advance :)

**Comment by VikashKothary** [comment]
Hi @Flinti07, 

I apologies for this issue, I was in the process of improving the CLI and happened to break its backward compatibility. You can see that discussion here: https://github.com/ankicommunity/anki-sync-server/issues/128.

TL;DR; It should be `python3 -m ankisyncd_cli`, but it has yet to be tested.
